cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -73,6 +73,13 @@ function Install-Binary($install_args) {
   {%- endfor %}
   }
 
+  {%- for updater in updaters %}
+  $platforms["{{ updater.target_triple }}"]["updater"] = @{
+    "artifact_name" = "{{ updater.id }}"
+    "bin" = "{{ updater.binary }}"
+  }
+  {%- endfor %}
```

Comment:
...i suppose that is the easiest way. makes me wonder if this should just be in the "artifacts" input above though

Reply:
Yeah, I wondered about that. In practice doing that had turned out to be a bit less convenient than I'd like, so this was easier for now.